heat/core/manipulations.py

```diff
@@ -830,10 +842,10 @@ def sort(a, axis=None, descending=False, out=None):
         for idx in np.ndindex(local_sorted.shape[1:]):
             idx_slice = [slice(None)] + [slice(ind, ind + 1) for ind in idx]
 
-            send_count = send_vec[idx][rank]
+            send_count = send_vec.cpu()[idx][rank]
```

Comment:
does this _need_ to be cpu? or can it get any device?

Reply:
I don't know why it's there. send_vec is already on the cpu. cpu() can be removed